frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/modal-footer.tsx

```diff
@@ -42,57 +43,118 @@ export const BlockPoolModalFooter = (props: BlockPoolModalFooterProps) => {
     close();
   };
 
-  return state.poolStatus ? (
+  return (
```

Comment:
You probably need a mapper or add more stuff in existing PROGRESS_STATUS for the footer now.
Something like this:
https://github.com/openshift/console/blob/2cd7bac47f2f892a2cb55e9950987a269732affc/frontend/packages/ceph-storage-plugin/src/utils/common-ocs-install-el.tsx#L40-L42